lua.template.editorconfig

```diff
@@ -101,6 +101,7 @@ align_call_args = false
 
 align_function_params = true
 
+# false or always
 align_continuous_assign_statement = true
 
 align_continuous_rect_table_field = true
```

Comment:
this option also support true/always/false

Reply:
Updated the pull request with your information, even though `true` seemed to not have any effect for me.

```ini
# true/false or always
align_continuous_assign_statement = true
```